core/src/i18n/en.yaml

```
@@ -823,7 +837,11 @@ functions:
     unhealthy: Unhealthy
   title: Functions
   tooltips:
+    building: Waiting for the image to be build.
```

```suggestion
    building: Waiting for the image to be built.
```

But perhaps I'd shorten it to `Waiting for the image to build`.